src/server/cell_labeling_app/endpoints/user_authentication.py

```diff
@@ -60,6 +48,8 @@ def login():
 
     user = db.session.query(User).filter_by(id=request_data[
         'email']).first()
+    if user is None:
+        return 'failure'
```

Comment:
In this case, we should return non 200 response code. It should be 400 since it is a user error. You can do that by returning the tuple
```
400, 'failure'
```